modules/depend_on_what_you_use/0.1.0/presubmit.yml

```diff
@@ -0,0 +1,15 @@
+# We configure this as it seems to be mandatory. But instead of executing integration tests as intended,
+# we build a trivial target. We trust our integration tests in our CI to ensure everything is fine for a release.
+# Either way, our integration tests are basd on Python scripts and currently could not be executed by a bazel test
```

Comment:
You can also specify `run_targets` and even Shell commands in case that would help.